glambie/processing/processing_helpers.py

```diff
@@ -67,12 +74,69 @@ def filter_catalogue_with_config_settings(data_group: GlambieDataGroup,
     if data_group == GLAMBIE_DATA_GROUPS["demdiff_and_glaciological"]:
         datasets_trend = [d for d in datasets_trend if d.data_group != GLAMBIE_DATA_GROUPS["glaciological"]]
 
+    # 4 add additional combined datasets stated in configs to data group
+    additional_annual_datasets = get_additional_combined_datasets(
+        data_catalogue=data_catalogue_original, data_group=data_group,
+        region_config=region_config, type_of_information="annual")
+    additional_trend_datasets = get_additional_combined_datasets(
+        data_catalogue=data_catalogue_original, data_group=data_group,
+        region_config=region_config, type_of_information="trend")
+    # combined solution should have an asterix so that they are recognized in the plots
+    for ds in additional_annual_datasets:
+        ds.user_group = ds.user_group + "_*"
+    for ds in additional_trend_datasets:
+        if ds not in additional_annual_datasets:
+            ds.user_group = ds.user_group + "_*"
+
+    datasets_annual.extend(additional_annual_datasets)
+    datasets_trend.extend(additional_trend_datasets)
+    log.info('Including the following combined datasets to ANNUAL calculations: datasets=%s',
+             additional_annual_datasets)
+    log.info('Including the following combined datasets to TREND calculations: datasets=%s', additional_trend_datasets)
+
     data_catalogue_annual = DataCatalogue.from_list(datasets_annual, base_path=data_catalogue.base_path)
     data_catalogue_trend = DataCatalogue.from_list(datasets_trend, base_path=data_catalogue.base_path)
 
     return data_catalogue_annual, data_catalogue_trend
 
 
+def get_additional_combined_datasets(data_catalogue: DataCatalogue, data_group: GlambieDataGroup,
```

Comment:
That's cleaner. As a side note this function is unlikely to be used outside of this module. As such, it should probably start with an underscore. Given it's then an internal function only, there's less stress on detailing full doc strings (a few inline comments to describe any complicated is sufficient).